README.md

```diff
@@ -293,10 +287,25 @@ installed.
 1. [Build](#build) the changes and push them to some tag
 2. Use the new tag in your workflows:
 
-```diff
--uses: paritytech/pr-custom-review@1
-+uses: paritytech/pr-custom-review@2
-```
+    ```diff
+    -uses: paritytech/pr-custom-review@1
+    +uses: paritytech/pr-custom-review@2
+    ```
+
+### Deployment
```

Comment:
One inconsistent point: not all steps start with a capital letter

1. **C** reate ...

2. **o** ne ...